src/software/ai/navigator/path_planner/bang_bang_trajectory_1d.h

```diff
@@ -0,0 +1,218 @@
+#pragma once
+
+#include <array>
+
+#include "software/ai/navigator/path_planner/trajectory.hpp"
+
+static const unsigned int MAX_TRAJECTORY_PARTS = 4;
+
+class BangBangTrajectory1D : public Trajectory<double, double, double>
```

Comment:
this should follow the same convention below and do
`using Trajectory1D = Trajectory<double, double double>`

Reply:
I ended up removing `Trajectory2D` anyways and making it an abstract class instead. The main reason I was using `using` was because I was extending the same same `Trajectory2D` class in multiple locations. `Trajectory1D` is currently just used here.